completions.go

```diff
@@ -117,22 +117,31 @@ type CompletionOptions struct {
 	HiddenDefaultCmd bool
 }
 
+// CompletionFunc is a function that provides completion results.
+type CompletionFunc func(cmd *Command, args []string, toComplete string) ([]string, ShellCompDirective)
+
 // NoFileCompletions can be used to disable file completion for commands that should
 // not trigger file completions.
+//
+// This method satisfies [CompletionFunc] and can be used with [Command.RegisterFlagCompletionFunc]
```

Comment:
How about changing this line to become:
```
// This method satisfies [CompletionFunc].
// It can be used with [Command.RegisterFlagCompletionFunc] and for [Command.ValidArgsFunction].
```